### ISSUE TITLE
Connect this thing to the cloud

### ISSUE THREAD

**Issue by LeanderK** [intellimate/Izou#19]
Everything it better with a cloud! And Bluetooth! This is the next big thing and every serious project needs a cloud! Cloud!

To make it possible for a normal person to use this project we should create a cloud-based website where you can manage Izou.

It would have the following features:
- a simple marketplace to add/remove AddOns to your instance of Izou.
- provide a mechanism to autoupdate AddOns
- make it possible to change settings using HTML
- manage Izou (updating Izou etc.)

Because this proposal is rather huge, i would suggest the following approach:
- clearly define the features
- make them as simple as possible
- Replace every resource intensive feature with a more simple placeholder if possible (even if it has some real drawbacks. But implement it in a way to make a future upgrade as simple as possible

The goal of this proposal is to:
- create a web interface for Izou, which can be used to manage Izou & AddOns
- define a standart to distribute AddOns

**Comment by jundl77** [comment]
+1

**Comment by jundl77** [comment]
deffinitaly the next step, but first (at least on my side) I want to get every thing working on v1.0, this should be our aim for v2.0 I think

**Comment by LeanderK** [comment]
I don't think this would require much changes for Izou in regards how Izou interacts with Addons. So I propose not to tie this thing up with the 2.0 release
